lib/handlers/buildModels.js

```diff
@@ -37,93 +152,13 @@ function buildModelsWrapper(modelBucket) {
             log = logHelpers.getDefault();
         }
 
+        partialModel = _.partial(modelFetcher, req, res);
```

Comment:
I'm not binding the `this`, just partially applying the first two parameters. This really is much cleaner.